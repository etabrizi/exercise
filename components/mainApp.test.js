import * as React from 'react';
import MainApp from './mainApp';
import renderer from 'react-test-renderer';

const Props = {
    work: [{}, {}, {}]
}

describe('MainApp component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<MainApp {...Props} />)
            .toJSON();
        expect(tree).toMatchSnapshot()
    });
});   
