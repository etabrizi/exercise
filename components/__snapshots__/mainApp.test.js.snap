// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainApp component renders correctly 1`] = `
<div
  className="jsx-2626450567"
  id="main-container"
>
  <p
    className="loading"
  >
    ..Sorry, there has been an error..
  </p>
</div>
`;
